index.html

```diff
@@ -0,0 +1,331 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/085384e239.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&family=Roboto:wght@500;700&display=swap"
+        rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link
+            href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@500;700&display=swap"
+            rel="stylesheet">
+</head>
+
+<body>
+    
+    <!----------navbar setion------------------->
+    <header>
+    <nav class="nav-container">
+        <a class="johnd" href="">John Doe</a>
+        <ul class="items-menu">
+            <li><a href="">Portfolio</a></li>
+            <li><a href="">About</a></li>
```

Comment:
- [ ] As stated 😃 by the previous reviewer kindly try to fix the overflow of the body because I see that there is a scroll in the horizontal, so kindly fix that to make your design attractive for the visitors😊

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/213722874-e99bc650-c937-44da-b9ff-e3b3abfc321b.png)|    ![image](https://user-images.githubusercontent.com/83360636/213722965-56625ce2-f628-4add-bb42-b43ed26a7bda.png)   |